data_steward/utils/participant_summary_requests.py

```diff
@@ -104,4 +124,56 @@ def get_deactivated_participants(project_id, columns):
     df = pandas.DataFrame(deactivated_participants,
                           columns=deactivated_participants_cols)
 
-    return df
+    # Converts column `suspensionTime` from string to timestamp
+    for column in deactivated_participants_cols:
+        if column == 'suspensionTime':
+            df[column] = pandas.to_datetime(df[column])
+
+    # Transforms participantId to an integer string
+    df['participantId'] = df['participantId'].apply(participant_id_to_int)
+
```

Comment:
You could use this to do the renaming:
```suggestion
    # rename participantId column to person_id
    df = df.rename(columns={'participantId': 'person_id'})
    
```

Reply:
Added 👍🏾